components/experimental/src/measure/category/length.rs

```diff
@@ -0,0 +1,38 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+
+use super::category;
+use crate::measure::measureunit::MeasureUnit;
+// use crate::measure::provider::si_prefix::{Base, SiPrefix};
+// use crate::measure::provider::single_unit::SingleUnit;
+// use smallvec::smallvec;
+// use alloc::vec;
+
+
+impl category::Length {
+    pub fn meter() -> &'static MeasureUnit {
+        // static METER: MeasureUnit = MeasureUnit {
+        //     single_units: vec![SingleUnit {
```

Comment:
@sffc , short_vec can be used.

@robertbastian , we can use `meter` then call `square` for more data.